src/main/java/org/sonarsource/auth/github/GitHubIdentityProvider.java

```diff
@@ -149,9 +150,16 @@ private boolean isOrganizationsMember(Token accessToken, String login) {
 
   /**
    * Check to see that login is a member of organization.
-   * https://developer.github.com/v3/orgs/members/#response-if-requester-is-an-organization-member-and-user-is-a-member
+   *
+   * A 204 response code indicates organization membership.  302 and 404 codes are not treated as exceptional,
+   * they indicate various ways in which a login is not a member of the organization.
+   *
+   * @see <a href="https://developer.github.com/v3/orgs/members/#response-if-requester-is-an-organization-member-and-user-is-a-member">GitHub members API</a>
    */
   private boolean isOrganizationMember(Token accessToken, String organization, String login) {
+    int membership_code = 204;
+    List<Integer> unexceptional_codes = Arrays.asList(membership_code, 302, 404);
```

Comment:
Please replace 204 by java.net.HttpURLConnection.HTTP_NO_CONTENT, 302 by HTTP_MOVED_TEMP, and 404 by  HTTP_NOT_FOUND.